packages/core/src/services/aws/CostAndUsageTypes.ts

```diff
@@ -53,6 +53,7 @@ export const HDD_USAGE_TYPES: string[] = [
   'GlacierByteHrs', // Glacier
   'Aurora:BackupUsage', // Aurora back up in S3
   'RDS:ChargedBackupUsage', // RDS Backup in S3
+  'USW2-RDS:ChargedBackupUsage', // RDS Backup in S3
```

Comment:
Hi @pwithnall -- thanks for the PR and apologies for the delayed response. This usage type should be handled by the string above it: `RDS:ChargedBackupUsage`. This is because in the CostAndUsageReports class we check that the string ends with any of the options in this array: `this.endsWithAny(HDD_USAGE_TYPES, usageType)`

Let me know if I'm missing something though and it's not being handled for you!

Reply:
I put this PR together because the server was printing a warning about `USW2-RDS:ChargedBackupUsage` not being handled, so something’s amiss. I followed the instructions in the documentation about handling those warnings to put this PR together. However, I haven’t tested it, so it could be the wrong fix.

Reply:
Hi @pwithnall - would you be able to pull the latest from the trunk branch and re-test and see if the warning is still showing? If it is, could you also log the entire `costAndUsageReportRow`, but changing line 212 of the `CostAndUsageReport` class:
`
this.costAndUsageReportsLogger.warn(
        `Unexpected usage type for storage service: ${costAndUsageReportRow}`,
      )`

Reply:
OK, I’ll give that a go when I get a chance in the next week or two, thanks.

Reply:
I dug into it a bit more and actually chased down the warning message, and it’s not a `CostAndUsageTypes` thing at all. I’ve updated the PR with a tested fix.